src/System.ValueTuple/tests/ValueTuple/UnitTests.cs

```diff
@@ -29,7 +29,7 @@ private class ValueTupleTestDriver<T1, T2, T3, T4, T5, T6, T7, T8, T9, T10>
         internal ValueTupleTestDriver(params object[] values)
         {
             if (values.Length > 10)
-                throw new ArgumentOutOfRangeException(nameof(values), "You must provide at most 10 values");
```

Comment:
>(nameof(values), [](start = 53, length = 16)

Is this change because we need to have the test written the same way for the desktop which using older version of the compiler?